language/analyze_v2/analyze_v2.go

```diff
@@ -0,0 +1,122 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Command analyze performs sentiment, entity, and classfication
+// on a string of text via the Cloud Natural Language API.
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/golang/protobuf/proto"
+
+	// [START language_imports]
+	language "cloud.google.com/go/language/apiv2"
+	"cloud.google.com/go/language/apiv2/languagepb"
+	// [END language_imports]
+)
+
+func main() {
```

Comment:
issue: don't use a CLI for samples.

See Samples Style Guide:
https://googlecloudplatform.github.io/samples-style-guide/#no-cli

Reply:
Done.